lively.morphic/world.js

```diff
@@ -47,9 +49,19 @@ export class World extends Morph {
           const { showsUserFlap } = resource(document.location.href).query();
           if (typeof showsUserFlap !== 'undefined') bool = showsUserFlap;
           this.setProperty('showsUserFlap', bool);
-          this.whenReady()
-            .then(() => System.import('lively.user/morphic/user-ui.js'))
-            .then(userUI => userUI.UserUI[bool ? 'showUserFlap' : 'hideUserFlap'](this));
+          System.import('lively.user/morphic/user-ui.js')
```

Comment:
Why doesn't this wait on this.whenReady() anymore?

Reply:
It does, see comment below

Reply:
Because we don't need to wait for it to be done with rendering, but we for the topbar to be loaded